src/core/Buttons/SplitButton/SplitButton.tsx

```diff
@@ -47,66 +51,74 @@ export type SplitButtonProps = {
  */
-export const SplitButton = (props: SplitButtonProps) => {
-  const {
-    onClick,
-    menuItems,
-    className,
-    menuPlacement = 'bottom-end',
-    styleType = 'default',
-    size,
-    children,
-    style,
-    title,
-    ...rest
-  } = props;
+export const SplitButton: SplitButtonComponent = React.forwardRef(
+  (props, forwardedRef) => {
+    const {
+      onClick,
+      menuItems,
+      className,
+      menuPlacement = 'bottom-end',
+      styleType = 'default',
+      size,
+      children,
+      style,
+      title,
+      ...rest
+    } = props;
 
-  useTheme();
+    useTheme();
 
-  const [isMenuOpen, setIsMenuOpen] = React.useState(false);
+    const [isMenuOpen, setIsMenuOpen] = React.useState(false);
 
-  const [menuWidth, setMenuWidth] = React.useState(0);
-  const ref = React.useRef<HTMLSpanElement>(null);
+    const [menuWidth, setMenuWidth] = React.useState(0);
+    const ref = React.useRef<HTMLSpanElement>(null);
 
-  React.useEffect(() => {
-    if (ref.current) {
-      setMenuWidth(ref.current.offsetWidth);
-    }
-  }, [children, size]);
+    React.useEffect(() => {
+      if (ref.current) {
+        setMenuWidth(ref.current.offsetWidth);
+      }
+    }, [children, size]);
 
-  return (
-    <span
-      className={cx(className, 'iui-button-split-menu', {
-        'iui-disabled': props.disabled,
-      })}
-      style={style}
-      title={title}
-      ref={ref}
-    >
-      <div>
-        <Button styleType={styleType} size={size} onClick={onClick} {...rest}>
-          {children}
-        </Button>
-      </div>
-      <div>
-        <DropdownMenu
-          placement={menuPlacement}
-          menuItems={menuItems}
-          style={{ minWidth: menuWidth }}
-          onShow={React.useCallback(() => setIsMenuOpen(true), [])}
-          onHide={React.useCallback(() => setIsMenuOpen(false), [])}
-        >
-          <IconButton
+    return (
+      <span
+        className={cx(className, 'iui-button-split-menu', {
+          'iui-disabled': props.disabled,
+        })}
+        style={style}
+        title={title}
+        ref={ref}
+      >
+        <div>
+          <Button
             styleType={styleType}
             size={size}
-            disabled={props.disabled}
+            onClick={onClick}
+            ref={forwardedRef}
+            {...rest}
           >
-            {isMenuOpen ? <SvgCaretUpSmall /> : <SvgCaretDownSmall />}
-          </IconButton>
-        </DropdownMenu>
-      </div>
-    </span>
-  );
-};
+            {children}
+          </Button>
+        </div>
+        <div>
+          <DropdownMenu
+            placement={menuPlacement}
+            menuItems={menuItems}
+            style={{ minWidth: menuWidth }}
+            onShow={React.useCallback(() => setIsMenuOpen(true), [])}
+            onHide={React.useCallback(() => setIsMenuOpen(false), [])}
+          >
+            <IconButton
```

Comment:
Split button `IconButton` is not rendered as anchor.

Reply:
But yeah, that doesn't make sense if it would. Irrelevant comment, just observation, sorry.